src/wallet/rpcdump.cpp

```diff
@@ -1522,6 +1522,7 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
                                     {"internal", RPCArg::Type::BOOL, /* default */ "false", "Stating whether matching outputs should be treated as not incoming payments (also known as change)"},
                                     {"watchonly", RPCArg::Type::BOOL, /* default */ "false", "Stating whether matching outputs should be considered watched even when not all private keys are provided."},
                                     {"label", RPCArg::Type::STR, /* default */ "''", "Label to assign to the address, only allowed with internal=false"},
+                                    {"keypool", RPCArg::Type::BOOL, /* default */ "false", "Stating whether imported public keys should be added to the keypool for when users request new addresses. Only allowed when wallet private keys are disabled"},
```

Comment:
15426: from PR description:
>Option was added in 14075 but not documented there

we don't have 14075 yet

Reply:
Thanks, and it turned out that 14075 needs [14021](https://github.com/bitcoin/bitcoin/pull/14021). As for 14021, it is marked as `VM_Stage` on the spreadsheet. Can I try to work on 14021 or should I wait that to be merged?